microreactors/hpmr_assembly/tests

```diff
@@ -0,0 +1,25 @@
+[Tests]
+  [generate_mesh]
+    type = RunApp
+    input = mesh.i
+    cli_args = '--mesh-only mesh.e'
+    recover = False # No recover for --mesh-only
+    executable_pattern = 'dire_wolf*'
```

Comment:
can sockeye not run these inputs?

rn you re failing direwolf tests. You d need to update the DW submodule

Reply:
It's using a material that's in Bison. And yeah, I was waiting for a DireWolf merge at the time - I'll update now.